.github/workflows/ci_webui.yaml

```diff
@@ -56,4 +60,26 @@ jobs:
           languages: javascript
 
       - name: Run CodeQL analysis
+        id: codeql
+        continue-on-error: true
         uses: github/codeql-action/analyze@v3
+
+      - name: Check for failures
+        if: always()
+        run: |
+          if [[ "${{ steps.lint.outcome }}" != "success" ]]; then
```

Comment:
I initially checked that `"${{ steps.lint.outcome }}" == "failure"`, but this current code seemed more resilient since it will fail if the above checks are renamed. In other words, this current code fails closed, whereas directly checking against an outcome of `failure` fails open.